applications/bluesky.md

```diff
@@ -0,0 +1,226 @@
+
+## Development Status 
+
+- references to conversations you might have had related to building this template: convosations in private chats 
+
+## Development Roadmap :nut_and_bolt:
+
+This section should break the development roadmap down into milestones and deliverables. Since these will be part of the agreement, it helps to describe _the functionality we should expect in as much detail as possible_, plus how we can verify and test that functionality. Whenever milestones are delivered, we refer to this document to ensure that everything has been delivered as expected.
+
+### Overview
+
+- **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
+	- 6 months
+- **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
+	- 3+ FTE
+- **Total Costs:** Requested amount in USD for the whole project (e.g. 12,000 USD). Note that the acceptance criteria and additional benefits vary depending on the level of funding requested. This and the costs for each milestone need to be provided in USD.
+	- 150,000 USD
+
+### Milestone 1 — Technical showcase example implemented in Ink! Macros and Substrate custom pallets for Ink! / Substrate / Polkadot novice. Publish Article 1/3 
+
+- **Estimated duration:** 2 month
+- **FTE:**  2,3
+- **Costs:** 40,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | Define use case | define a contrived use case that requires 1) business logic 2) reusable components 3) features needing smart contract upgrades 4) performance and low gas fees |
+| 2. | define list of Ink! Macros | that captures reusable logic to simplify code |
+| 3. | Substrate modules | implement substrate custom pallets corresponding to above macros |
+| 4. | Smart contracts | Write Ink! smart contract with macros integrated with custom pallets to fulfil use case business functions |
+| 5. | End-to-end integration | Organically integrate all pieces together to ensure working end-to-end |
+| 6. | Deployment | Deploy to a Polkadot smart contract parachain, say Shiden, etc to also explore interoperability |
+| 7. | Articles | We will publish article 1 of 3 walking through the technical showcase focusing on Ink! macros and substrate pallets. Can publish the blog in Medium in English, or recommended medium to target designated audiences |
+
+### Milestone 2 —  On-chain Ink! smart contract upgrades, compared to Solidity upgrades. Publish Article 2/3   
+
+- **Estimated Duration:** 2 month
+- **FTE:**  2,3
+- **Costs:** 50,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | Upgrade Ink! smart contracts due to feature changes | Demonstrate Upgradability via ink!'s built-in upgradeable smart contract features to upgrade the smart contract code without having to migrate the data or interrupt the user experience |
```

Comment:
How will this differ from the [example here](https://github.com/paritytech/ink-examples/tree/main/upgradeable-contracts)?

Reply:
While that example is on forward-calls and set-code-hash, this showcase will mostly focus on data migration from old to new contract, Ink! built-in tools and libraries for managing contract upgrades, and seamless experiences as compared to Solidity.